src/TMapLabel.h

```diff
@@ -36,19 +36,25 @@
 class TMapLabel
 {
 public:
+    // Try and deduce whether this label has data:
+    bool isNull() const {return pos.isNull() && size.isNull() && text.isEmpty() && pix.isNull(); }
+    QByteArray base64EncodePixmap() const;
 
     QVector3D pos;
     QSizeF size;
     QSizeF clickSize;
     QString text;
+#if (QT_VERSION) >= (QT_VERSION_CHECK(5, 14, 0))
```

Comment:
We (or rather Qt) has switched to `QColorConstants` in Qt 5.14 and we have already started to use this alternative elsewhere and since I was in this area I've tweaked this at the same time.